### PR TITLE
updated snapdragon debrick information

### PR DESCRIPTION
**Pull request by RomanBapst**
@nicovanduijn @DanielePettenuzzo @hamishwillee FYI

The information I added stems from [here](https://github.com/ATLFlight/ATLFlightDocs/blob/master/TroubleshootFAQ.md)

Signed-off-by: Roman <bapstroman@gmail.com>

### CODE REVIEW

File: en/flight_controller/snapdragon_flight_advanced.md [hunk 1]
```diff
@@ -249,6 +249,15 @@ If it still doesn't work, check [this answer on StackOverflow](http://askubuntu.
 
 ### Board doesn't start / is boot-looping / is bricked
```
Comment (hamishwillee) [inline en/flight_controller/snapdragon_flight_advanced.md]: We use first letter capitalisation on all headings - i.e. Board Doesn't Start / Board Boot-Looping / Board Bricked

(change everywhere)

**Comment by hamishwillee** [comment]
This looks OK at high level. The only concern is license - we have to attribute under CC.  Could we ask the copyright holders if it is OK to just repost here with re-attributing - ie fresh start here. Otherwise we'll need to add a comment for the section.

File: en/flight_controller/snapdragon_flight_advanced.md (hunk 1 above)
Comment (RomanBapst) [reply en/flight_controller/snapdragon_flight_advanced.md]: @hamishwillee Ok, are there any exceptions about this? I went through the guide and found several instances where this is not the case. Is is 'Forums and Chat' or 'Forums And Chat'?

File: en/flight_controller/snapdragon_flight_advanced.md [hunk 2]
```diff
@@ -249,6 +249,15 @@ If it still doesn't work, check [this answer on StackOverflow](http://askubuntu.
...
 
+The only solution here will be to try and update the Android/Linux image on your board.
+If you happen to have a [P2 board](#do-i-have-a-p1-or-p2-board), you should be able to reset the Snapdragon to the recovery image by starting up the Snapdragon while shorting the two pins next to where J3 is written (The two rectangular pins in-between the corner hole and the MicroSD card slot almost at the edge of the board.
+
+If that does not work you can try one of the following two options to put the board into fastboot mode, which will then enable you to update the Android/Linux image.
+
+If everything fails, you probably need to request help from intrinsyc.
```
Comment (hamishwillee) [inline en/flight_controller/snapdragon_flight_advanced.md]: Capitalize "intrinsyc" - proper noun

File: en/flight_controller/snapdragon_flight_advanced.md [hunk 3]
```diff
@@ -249,6 +249,15 @@ If it still doesn't work, check [this answer on StackOverflow](http://askubuntu.
 
 ### Board doesn't start / is boot-looping / is bricked
 
+The only solution here will be to try and update the Android/Linux image on your board.
+If you happen to have a [P2 board](#do-i-have-a-p1-or-p2-board), you should be able to reset the Snapdragon to the recovery image by starting up the Snapdragon while shorting the two pins next to where J3 is written (The two rectangular pins in-between the corner hole and the MicroSD card slot almost at the edge of the board.
```
Comment (hamishwillee) [inline en/flight_controller/snapdragon_flight_advanced.md]: Does the link "#do-i-have-a-p1-or-p2-board" exist on this page?

File: en/flight_controller/snapdragon_flight_advanced.md (hunk 1 above)
Comment (hamishwillee) [reply en/flight_controller/snapdragon_flight_advanced.md]: Yes, you don't capitalise the small connecting words unless they are at the beginning of the sentence - and, on, the, with, for, is etc. 

With respect to the license, perhaps we could simply add a note.

> **Note** This section of text originates from the [ATLFlightDocs](https://github.com/ATLFlight/ATLFlightDocs#snapdragon-flight-documentation)

There are a couple of other minor comments. Assuming you're happy with the above I can merge this and make the changes as needed (?)

**Comment by hamishwillee** [comment]
@RomanBapst What is status of this?

**Comment by hamishwillee** [comment]
@RomanBapst Is there any value to this still? If not, I plan to close it unmerged next time I come here. 

If you think it is still useful let me know, and I will merge with the minor tidy up suggested.

**Comment by hamishwillee** [comment]
I have decided to close this. No response from poster.

**Pull request closed without merging**